src/components/ListingResults/index.js

```diff
@@ -6,9 +6,9 @@ import cs from "./styles.module.css"
 function ListingResults({ listings }) {
   return (
     <div className={cs.listings}>
-      {listings.map(listing => (
-        <Listing key={listing.key} listing={listing}></Listing>
-      ))}
+      {listings
+        .map(listing => <Listing key={listing.key} listing={listing}></Listing>)
+        .flat()}
```

Comment:
yes - because when the Listing maps it may return multiple cards (i.e. `[[FinancialNeedCard, OtherCard], [FinancialNeedCard], [OtherCard]]`) it flattens them for rendering.